filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilder.scala

```diff
@@ -45,7 +47,7 @@ object BlobPathBuilder {
       val uri = parseURI(string)
       val storageAccount = parseStorageAccount(parseURI(endpoint))
       val hasContainer = uri.getPath().split("/").filter(!_.isEmpty()).headOption.contains(container)
```

Comment:
While I'm in here taking credit for IntelliJ's recommendations... can replace `.filter(...).headOption` with `.find(...)`.

Reply:
Oi, it seems Metals is letting me down at bit, I'll try working with intellij more.